### ISSUE TITLE
Typing

### ISSUE THREAD

**Issue by karlvr** [FGRibreau/node-truncate#8]
Would you like to add types to the project? We whipped up these types and would be very happy if you included them in your library!

Also it looks like the `keepImageTag` option is documented but not implemented?

```typescript
declare module 'truncate' {
	interface TruncateOptions {
		keepImageTag: boolean
		ellipsis?: string
	}

	export default function truncate(string: string, maxLength: number, options?: TruncateOptions): string
}
```

**Comment by FGRibreau** [comment]
Indeed you are right (about the imageTag) please open a PR for the types and remove the comment as well :)

Thanks :+1:

**Comment by karlvr** [comment]
@FGRibreau thank you, I've made a PR #9

**Comment by FGRibreau** [comment]
Released in v2.1.0, thanks!
